src/typescript/01-intro-to-typescript/intro-to-typescript.md

```diff
@@ -50,4 +52,8 @@ TypeScript uses static typing, which allows us to specify what type a variable s
 
 In this guide we will teach you about TypeScript concepts and have an environment for you to practice in. Running the tests will catch any problems if the code was not written correctly or as expected. Pay close attention to how you name and save the files.
 
-When we give you a command to run to verify your work it will look for a file named respectively. If you run into issues you can look at the package.json file to see which file the command listed is looking for.
\ No newline at end of file
+When we give you a command to run to verify your work it will look for a file named respectively. If you run into issues you can look at the package.json file to see which file the command listed is looking for.
```

Comment:
This is no longer true, see [Exporting and Importing](https://www.bitovi.com/academy/learn-typescript/exporting-and-importing.html) for how we’re going to approach this.

Speaking of which, could you ask Christopher for help with getting that page and [Setting Up Your Environment](https://www.bitovi.com/academy/learn-typescript/setting-up-your-environment.html) published through HubSpot please?